interactive-bots/helloworld/Smehnov/notes.md

```diff
@@ -0,0 +1 @@
+I had many problems while I did this task, but I solved them thanks to mentors
```

Comment:
* I suggest you to end the sentence with a period (.). 
* You could be even more descriptive about your problems and how you solved them. This is not necessary though! :)